doc/page-features.h

```diff
@@ -151,4 +151,20 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section IPCServerRegistrationFromCpp Registration of an IPC server from C++ code
+Instead of enabling IPC in the QML UI code (with \c IPC.enabled: \c true), it is possible to register an IPC server in the C++ implementation.
+The following code snippet shows how this can be achieved :
+\code
+#include "MyInterface.h"
+#include "MyInterfaceIPCAdapter.h"
+
+void registerIPCServer(MyInterface& objectImplementingMyInterface)
+{
+   auto ipcAdapter = new MyInterfaceIPCAdapter();
+   ipcAdapter->setService(objectImplementingMyInterface);
+   ipcAdapter->registerService();
+}
```

Comment:
Should we mention that this would leak memory and in real code the user would have to make sure to delete the ipcAdapter again?

Reply:
As  its a code snippet show casing how this use case  can be achieved  , lets keep this   to the user's discretion  to take care of life time of object  .